src/modules/situations/situation-message-box.tsx

```diff
@@ -96,15 +104,21 @@ export const MessageBox = ({
   message,
   title,
   onClick,
+  borderRadius = true,
 }: MessageBoxProps) => {
   const { static: staticColors } = useTheme();
   const { t } = useTranslation();
   const backgroundColorStyle = {
     backgroundColor: staticColors['status'][type].background,
   };
 
+  const borderRadiusStyle = borderRadius ? style.borderRadius : '';
+
   return (
-    <div className={style.container} style={backgroundColorStyle}>
+    <div
+      className={and(style.container, borderRadiusStyle)}
```

Comment:
Not a big deal in this case, but IIRC there's an `andIf` utility function that can be used in case any of the classes should be conditionally applied.

Reply:
True, updated now 👍